backend/src/election/api.rs

```diff
@@ -129,8 +128,13 @@ pub async fn election_create(
 pub struct ElectionDefinitionValidateRequest {
```

Comment:
This struct should be named differently to reflect that it also takes a candidate list, not just the election definition EML.

Same for the comment on L142-143 which only mentions the election definition.